File: proyecto movil/MYDAY2/app/src/main/java/com/example/myday/Usuario.java
package com.example.myday;

public class Usuario {
    private String nombreUsuario;

    private String email;
    private String contrasena;

    public Usuario(String nombreUsuario,String email,String contrasena){
        this.nombreUsuario = nombreUsuario;

        this.email = email;
        this.contrasena = contrasena;
    }
    public void setNombreUsuario(String nombreUsuario) {
        this.nombreUsuario = nombreUsuario;
    }



    public void setEmail(String email) {
        this.email = email;
    }

    public void setContrasena(String contrasena) {
        this.contrasena = contrasena;
    }

    public String getNombreUsuario() {
        return nombreUsuario;
    }



    public String getEmail() {
        return email;
    }

    public String getContrasena() {
        return contrasena;
    }


}

File: proyectoDI/MYDAY/MYDAY/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MYDAY
{
    public class Usuario
    {
        public string nombreUsuario { get; set; }
        public string contrasena { get; set; }
        public string email { get; set; }
        
        public Usuario(string nombreUsuario, string contrasena, string email)
        {
            this.nombreUsuario = nombreUsuario;
            this.contrasena = contrasena;
            this.email = email;
            
        }
    }
}
